common/v2/theme/constants.ts

```diff
@@ -85,6 +85,7 @@ export const LINE_HEIGHT = {
 };
 
 export const SPACING = {
+  NONE: '0',
```

Comment:
I'm not sure this has an utility since 0 will always be 0 :)

Reply:
Yeah, good point